src/core_plugins/kibana/public/dashboard/dashboard.js

```diff
@@ -286,7 +287,43 @@ app.directive('dashboardApp', function ($injector) {
         }).catch(notify.fatal);
       };
 
+      $scope.showFilterBar = () => filterBar.getFilters().length > 0 || !$scope.fullScreenMode;
+      let onRouteChange;
+      const setFullScreenMode = (fullScreenMode) => {
+        $scope.fullScreenMode = fullScreenMode;
+        dashboardState.setFullScreenMode(fullScreenMode);
+        chrome.setVisible(!fullScreenMode);
+        $scope.$root.$broadcast('globalNav:update');
```

Comment:
yea, I figured the global navigation did update, so it wasn't that bad, though I hate those broadcasts.  I can do `$window.dispatchEvent(new Event('resize'));` to trigger the re-render of the grid, which should also fixe the bug @nreese  discovered.  How do you feel about that?  I don't think there is really a good way to tell gridster to resize directly.  Or maybe I'm missing a more obvious solution...